paasta_tools/marathon_tools.py

```diff
@@ -1474,12 +1480,12 @@ def is_old_task_missing_healthchecks(task: MarathonTask, app: MarathonApp) -> bo
 
 def get_num_at_risk_tasks(app: MarathonApp, draining_hosts: List[str]) -> int:
     """Determine how many of an application's tasks are running on
-    at-risk (Mesos Maintenance Draining) hosts.
+    at - risk(Mesos Maintenance Draining) hosts.
 
-    :param app: A marathon application
-    :param draining_hosts: A list of hostnames that are marked as draining.
+    : param app: A marathon application
+    : param draining_hosts: A list of hostnames that are marked as draining.
                            See paasta_tools.mesos_maintenance.get_draining_hosts
-    :returns: An integer representing the number of tasks running on at-risk hosts
+    : returns: An integer representing the number of tasks running on at - risk hosts
```

Comment:
what's up with these changes? overzealous find and replace?

Reply:
No idea. I'll kill it.